### ISSUE TITLE
[Enhancement] release Releases on Repository of Chocolatey packages

### ISSUE THREAD

**Issue by GoldenHashtag** [Angelelz/WinHelloUnlock#15]
(Yeah I've just copied another issue)
Is your feature request related to a problem? Please describe.
Hello, this is not an issue in itself. No. I'm always frustrated when to find & download and upgrade plgx and manually compare its integrity for security reasons.
Describe the solution you'd like
Just an suggestion.
Would be nice to have a auto update feature for this primary security applications plugin from trusted source. The sha256 hash of PLRGX can be verified by installation automatically optionally if specified. For the lazy ones :) and to fix incompatibility issue with future of keepass version updates with plugin by getting all in one place for example.
I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contains a script to URL of github latest release asset.
Additional context
If you may want to take a look here if you don't know already:
keepass plugins packages
(There are many plugins and packages count is growing. )

https://github.com/sirAndros/KeePassWinHello/issues/22

**Comment by Angelelz** [comment]
This is way ahead, very close to wontfix. I will look into it later

**Comment by BananaAcid** [comment]
This would be great. I am using chocolatey all the time. Install and manage KeePass using it.

**Comment by BananaAcid** [comment]
F*** - manually updating the plugin to the new update onto 430 devices (since the approving process for automagically managing software is only planed on top of chocolatey).

**Comment by BananaAcid** [comment]
BTW to resolve this enhancement, you might use this files:
```
https://github.com/sirAndros/KeePassWinHello
/Deploy-Plugin.ps1
/Pack-Plugin.ps1
/Chocolatey/* (nuspec!)
```

nearly 1 to 1, but fixingdescriptions and author. (Since it is about installing to he same target and using GitHub as well)
